scripts/foglamp

```diff
@@ -121,7 +123,7 @@ foglamp_start() {
 
         "2")
             #FogLAMP not running
-            echo -n "Starting FogLAMP."
+            echo -n "Starting FogLAMP ${FOGLAMP_VERSION}."
```

Comment:
`Starting FogLAMP 1.1.2......`

better would be to echo `Starting FogLAMP v1.1.2 ......`

Reply:
Added v${FOGLAMP_VERSION}